sdk/spring/azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/cosmos/CosmosAutoConfigurationTest.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.microsoft.azure.spring.autoconfigure.cosmos;
+
+
+import com.azure.cosmos.ThrottlingRetryOptions;
+import com.azure.cosmos.implementation.ConnectionPolicy;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Ignore
+public class CosmosAutoConfigurationTest {
```

Comment:
What's this test for?

Reply:
it's used for testing properties setting of `ConnectionPolicy`